backend/core/src/installed_game.rs

```diff
@@ -43,7 +43,7 @@ impl InstalledGame {
 		// We ignore games that don't have an extension.
 		let extension = path.extension()?.to_str()?;
 
-		if !VALID_EXTENSIONS.contains(&extension) {
+		if !VALID_EXTENSIONS.contains(&extension.to_ascii_lowercase().as_str()) {
```

Comment:
Is there any particular reason to use to_ascii_lowercase here instead of to_lowercase? Asking because maybe there's a reason to use this in other places where I used to_lowercase before.

Reply:
Nothing special here, can switch to `to_lowercase`.